### ISSUE TITLE
App process is running but no ui is visible

### ISSUE THREAD

**Issue by sdillen** [RocketChat/Rocket.Chat.Electron#1327]
## My Setup
- Operating System: Windows 10 Version 1903
- App Version: 2.15.5
- Installation type: exe

- [x] I have tested with the latest version
- [ ] I can simulate the issue easily

## Description
In our company we are rolling out the application on every Windows 10 device. This bug is coming up on a regular basis.

### Current Behavior
The process of the application is running but the ui is not present. After killing the process and deleting this `C:\Users\my-user\AppData\Roaming\Rocket.Chat` directory the application worked like expected.

### Expected Behavior
The process of the application is running and the ui is present

**Comment by brandonharshe** [comment]
Are you sure it's not hidden in the hidden icons tray? If so, this is default behavior, discussed in https://github.com/RocketChat/Rocket.Chat.Electron/issues/1104. Isn't AppData where the application is stored if it's installed for the user only (not all users on the computer)?

**Comment by sdillen** [comment]
Yeah I'am sure that it's not hidden in the system tray. As I said the process is running but I get no visual confirmation about it.
The installation is for all users on the computer so it's not stored in AppData.
